lib/helpers/setupNetlifyFunctionForPage.js

```diff
@@ -14,24 +14,25 @@ const setupNetlifyFunctionForPage = (filePath) => {
   const functionDirectory = join(NETLIFY_FUNCTIONS_PATH, functionName);
 
   // Copy function template
-  copySync(
-    FUNCTION_TEMPLATE_PATH,
-    join(functionDirectory, `${functionName}.js`),
-    {
+  const functionTemplateCopyPath = join(
+    functionDirectory,
+    `${functionName}.js`
+  );
+  if (!existsSync(functionTemplateCopyPath)) {
+    copySync(FUNCTION_TEMPLATE_PATH, functionTemplateCopyPath, {
       overwrite: false,
       errorOnExist: true,
-    }
-  );
+    });
+  }
 
   // Copy page
-  copySync(
-    join(NEXT_DIST_DIR, "serverless", filePath),
-    join(functionDirectory, "nextJsPage.js"),
-    {
+  const nextPageCopyPath = join(functionDirectory, "nextJsPage.js");
+  if (!existsSync(nextPageCopyPath)) {
+    copySync(join(NEXT_DIST_DIR, "serverless", filePath), nextPageCopyPath, {
       overwrite: false,
       errorOnExist: true,
-    }
-  );
+    });
+  }
```

Comment:
You could just have changed `errorOnExist` to `false` :yum: 

But either way, I don't think this is the ideal solution. The pages `/test_me` and `/test/me` will both want try to create a function named `next_test_me`. At the moment, this will trigger an error and let the user know there is an issue. If we don't raise an error, this will go unnoticed. We can reduce the chance of collision with a hash at the end of the function (which we should!), but we can't guarantee non-collision, so personally, I think it's best to tackle this issue at the "root" (in /getStaticProps/setup.js) and make sure we never create a Netlify Function for the same page twice.

Reply:
i see that's a good point, thanks for callin out!